Cargo.toml

```diff
@@ -73,16 +73,22 @@ resolver = "2"
 # it will publish unbuildable releases to crates.io
 
 [workspace.lints.clippy]
-nursery = "allow"
-cargo = "allow"
-pedantic = "allow"
-restriction = "allow"
+nursery = { level = "allow", priority = -1 }
```

Comment:
Why the reduced priority here?

Reply:
It's a bug where Cargo can't tell these levels apart from the defaults unless you explicitly set a priority. It's recommended by a build warning to set it like this instead